css/base/responsive.css

```diff
@@ -0,0 +1,21 @@
+/* Add media queries breakpoints */
+@media only screen and(max-width: 379px) {
+}
+
+@media only screen and(max-width: 479px) {
+}
+
+@media only screen and(max-width: 575px) {
+}
+
+@media only screen and(max-width: 767px) {
+}
+
+@media only screen and(max-width: 991px) {
+}
+
+@media only screen and(max-width: 1199px) {
+}
+
+@media (min-width: 1500px) {
+}

```

Comment:
Bootstrap'o versija kuria ikėlei jau turi 5 breakpont'us, ar tikrai reikia dar 7 papildomu?

Reply:
Tai darom tik su bootsrap tuomet?

Reply:
Aš asmeniškai daryčiau be bootstrapo, bet jai visi nutarėm ji naudoti, tai siulau ir pasilikti ties jo breakpointais, kad nebūtu, jog pas visus skirtunguose breakpointuose puslapis reaguos:)

Reply:
Aš asmeniškai taip pat daryčiau be boostrap, neįdomu taip. Tai gal be jo darome?